lib/_http_server.js

```diff
@@ -515,15 +515,24 @@ function connectionListener(socket) {
       }
     }
 
-    if (req.headers.expect !== undefined &&
-        (req.httpVersionMajor == 1 && req.httpVersionMinor == 1) &&
-        continueExpression.test(req.headers['expect'])) {
-      res._expect_continue = true;
-      if (EventEmitter.listenerCount(self, 'checkContinue') > 0) {
-        self.emit('checkContinue', req, res);
+    if (typeof req.headers.expect !== 'undefined' &&
```

Comment:
Just a nit, but checking directly against `undefined` (`req.headers.expect !== undefined`) instead of using `typeof` is preferred, when compared to the rest of the code base.